lib/webvalve/manager.rb

```diff
@@ -66,7 +66,13 @@ def allowing?
     end
 
     # @api private
-    def reset
+    def reset!
+      clear!
+      setup
+    end
+
+    # @api private
+    def clear!
```

Comment:
If you know the internals really well, this is gonna feel weird, because it calls WebMock.reset! internally.

But if the whole point of a reset is to get back to the zero state, it feels like we want a "clear" that empties everything and a "reset" that gets you back to that "zero state" of pre-test, post-initializer setup. (It's like `db:reset`)